Source/WebInspectorUI/UserInterface/Views/GradientEditor.css

```diff
@@ -24,25 +25,13 @@
  */
 
 .gradient-editor {
+    --gradient-editor-controls-width:  237px;
```

Comment:
Style: should only have one space after `:`

NIT: I usually put CSS variables at the bottom of a declaration so that the actual things that affect style are shown first (similar to how we put member variables at the bottom of a `class` in C++)